data/database/40f6ed488e7aeef1d26202c8d3514ce8dab9993f9aee25201452598332a53aaa

```diff
@@ -0,0 +1 @@
+{"response":{"status":"ok","userTier":"internal","total":1,"content":{"elements":[],"id":"membership/2015/apr/27/the-spin-london-cycle-festival-save-mop","sectionId":"news","webTitle":"The Spin London cycle festival: save 50% on tickets for your sessions","references":[],"webPublicationDate":"2015-04-27T09:55:30Z","fields":{"internalComposerCode":"553a2089e4b095bced12fa24","membershipAccess":"members-only","isPremoderated":"false","shouldHideAdverts":"false","wordcount":"16","shortUrl":"http://gu.com/p/47zbv","commentCloseDate":"2015-05-04T09:55:30Z","standfirst":"<p>Spin London is a must for cycling fans. The festival runs at The Sorting Office on London’s New Oxford Street from 8 to 10 May. Our offer is valid for any session over the weekend</p>","linkText":"The Spin London cycle festival: save 50% on tickets for your sessions","liveBloggingNow":"false","body":"<p>To take up this offer, book via <a href=\"https://www.eventbrite.co.uk/e/spin-london-2015-the-urban-cycle-show-tickets-15618120218?discount=GUARDIAN\">this dedicated link</a>.</p> <h2><strong>Terms and conditions</strong></h2> <p>•Subject to availability.</p>","productionOffice":"UK","showInRelatedContent":"false","hasStoryPackage":"false","internalPageCode":"2281338","internalContentCode":"456481939","publication":"theguardian.com","trailText":"Spin London is a must for cycling fans. The festival runs at The Sorting Office on London’s New Oxford Street from 8 to 10 May. Our offer is valid for any session over the weekend ","allowUgc":"false","lastModified":"2015-04-27T11:43:45.043+01:00","headline":"The Spin London cycle festival: save 50% on tickets for your chosen sessions","isInappropriateForSponsorship":"false","commentable":"false","firstPublicationDate":"2015-04-27T10:55:30.360+01:00"},"webUrl":"http://www.theguardian.com/membership/2015/apr/27/the-spin-london-cycle-festival-save-mop","apiUrl":"http://content.guardianapis.com/membership/2015/apr/27/the-spin-london-cycle-festival-save-mop","sectionName":"News","tags":[{"id":"type/article","webTitle":"Article","type":"type","webUrl":"http://www.theguardian.com/articles","apiUrl":"http://content.guardianapis.com/type/article","references":[]}]},"storyPackage":[]}}
```

Comment:
@robertberry Should I be committing these files? Is this what's generated by `HttpRecorder` for TC?